### PR TITLE
Payment Admin Page + TemplateComponent

### PR DESCRIPTION
**Pull request by Raemon**
This creates a simple page for admins to view users who have a paymentEmail or paymentInfo, to make that easier to track.

It also adds a TemplateComponent which can be copied to get a minimal bare-bones component setup.

### CODE REVIEW

**Comment by Raemon** [comment]
No. (I did end up making some tweaks between initially posting it and now, but it is meant to be an initial small chunk of work that will be built on later, while also giving me some immediate value)

File: packages/lesswrong/lib/routes.ts [hunk 1]
```diff
@@ -711,6 +711,16 @@ addRoute(
...
+  {
+    name: 'paymentsAdmin',
+    path: '/payments/admin',
+    componentName: 'AdminPaymentsPage'
+  },
+  {
+    name: 'payments',
+    path: '/payments',
+    redirect: () => `/payments/admin`, // eventually, payments might be a userfacing feature, and we might do something else with this url
+  },
```
Comment (jpaddison3) [inline packages/lesswrong/lib/routes.ts:723]: These should be in a LW-specific branch of routes.

File: packages/lesswrong/components/common/TemplateComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+const TemplateComponentComponent = registerComponent('TemplateComponent', TemplateComponent, {styles});
+
+declare global {
+  interface ComponentTypes {
+    TemplateComponent: typeof TemplateComponentComponent
+  }
+}
+
```
Comment (jpaddison3) [inline packages/lesswrong/components/common/TemplateComponent.tsx:25]: Nice, this'll be useful, thanks!

**Review by jpaddison3 — COMMENTED** [review]
Not offering full review, as this is LW-only. But from my vantage point (haven't tested locally), this looks straightforward.

**Comment by Raemon** [comment]
Moved routes to LessWrong specific. (I'm not 100% sure this makes sense, I think there's a good chance having a payment system would be good for EA forum, but it does seem fine to leave it LW specific until it's more fleshed out and easier to discuss)

**Review by darkruby501 — APPROVED** [review]
Looks good. Would generally like to see a screenshot though.

**Comment by Raemon** [comment]
The screenshot would include user's semi-private paypal info (not maximally private but it didn't seem worth it given that it's also a pretty simple, admin-only page)

**Pull request merged**